src/Data/Char/Properties.agda

```diff
@@ -35,6 +37,9 @@ open import Agda.Builtin.Char.Properties
 ≈⇒≡ : _≈_ ⇒ _≡_
 ≈⇒≡ = toℕ-injective _ _
 
+≉⇒≢ : ∀ {x y} → ¬ (x ≈ y) → x ≢ y
```

Comment:
It'd be nice to define `_≉_` in the `Data.Char.Base` and then use it here?

Reply:
Won't that clash with setoid also defining it?

Reply:
We don't have any opened `Setoid` in scope here right? If you mean in general if we imported `Data.Char` with an opened `Setoid` in scope, then aren't we already getting clashes with `_≈_` so I'm not sure an extra one is problematic?